plugins/chat/assets/javascripts/discourse/pre-initializers/chat-plugin-api.js

```diff
@@ -163,6 +163,8 @@ export default {
               .lookup("service:chat-api")
               .sendMessage(channelId, {
                 thread_id: options.threadId,
+                post_ids: options.postIds,
+                topic_id: options.topicId,
```

Comment:
```suggestion
                context_post_ids: options.postIds,
                context_topic_id: options.topicId,
```

In comments we call these values "currently visible posts", or "context". Also in the message bus event they are grouped like this:

```ruby
{ 
  context: 
  { 
    post_ids: contract.post_ids, 
    topic_id: contract.topic_id
  } 
}
```

We may not group them here, but it would be good to use more descriptive names: `context_post_ids` and `context_topic_id`.

Reply:
hmmm that's an interesting idea, will do it :+1: